sdks/desktop/Makefile

```diff
@@ -0,0 +1,15 @@
+TOP=$(realpath $(CURDIR)/../..)
```

Comment:
I don't understand this dependency at all

Reply:
This is done to include the paths.mk file on the next line